### PR TITLE
[Script Debugging] Update For New Release 538

### PR DESCRIPTION
**Pull request by dianaqu**
- include script debugging testing for next release
- copy modify files to support next release

TODO: update to new sdk version after release

### CODE REVIEW

File: WebView2APISample/README.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ Clone the repository and open the solution in Visual Studio. WebView2 is already
 - Clone this repository
 - Open the solution in Visual Studio 2019**
 - Set the target you want to build (Debug/Release, x86/x64/ARM64)
-- Build the project file: `WebView2APISample.vcxproj`
+- Build the solution file: WebView2APISample.sln
```
Comment (peiche-jessica) [inline WebView2APISample/README.md:28]:
```suggestion
- Build the solution file: `WebView2APISamples.sln`
```

I will fix the copy in ADO too.

Comment (dianaqu) [reply WebView2APISample/README.md:28]: should this be `WebView2APISamples.vcxproj` instead?

Comment (peiche-jessica) [reply WebView2APISample/README.md:28]: If we want to scope to win32 sample project but the collective solution, we can just revert the change here
```suggestion
- Build the project file: `WebView2APISample.vcxproj`
```

Comment (dianaqu) [reply WebView2APISample/README.md]: I just run the script again. and this already got updated. should I still change it to sln?

Comment (peiche-jessica) [reply WebView2APISample/README.md]: Let's not worry about it then. Thanks!

**Pull request merged**